hi-ml/testhiml/testhiml/test_runner.py

```diff
@@ -113,6 +113,30 @@ def test_additional_aml_run_tags(mock_runner: Runner) -> None:
         assert "max_epochs" in mock_submit_to_azure_if_needed.call_args[1]["tags"]
 
 
+def test_additional_environment_variables(mock_runner: Runner) -> None:
+    model_name = "HelloWorld"
+    arguments = ["", f"--model={model_name}", "--cluster=foo"]
+    with patch.multiple(
+        "health_ml.runner",
+        submit_to_azure_if_needed=DEFAULT,
+        check_conda_environment=DEFAULT,
+        get_workspace=DEFAULT,
+        get_ml_client=DEFAULT,
+    ) as mocks:
+        with patch("health_ml.runner.Runner.run_in_situ"):
+            with patch("health_ml.runner.Runner.parse_and_load_model"):
+                with patch("health_ml.runner.Runner.validate"):
+                    with patch.object(sys, "argv", arguments):
+                        mock_container = create_autospec(LightningContainer)
+                        mock_container.get_additional_environment_variables = MagicMock(return_value={"foo": "bar"})
+                        mock_runner.lightning_container = mock_container
+                        mock_runner.run()
+        mocks["submit_to_azure_if_needed"].assert_called_once()
+        assert DEBUG_DDP_ENV_VAR in mocks["submit_to_azure_if_needed"].call_args[1]["environment_variables"]
```

Comment:
[Minor] Maybe extract `mocks["submit_to_azure_if_needed"].call_args[1]["environment_variables"]` to a variable, to simplify?